### PR TITLE
Add MemProcFS

### PR DESCRIPTION
**Pull request by naacbin**
Add of MemProcFS, a tool to analyze memory images. Also, add a required dependency, dokan, a driver for virtual file system.

Code MemProcFS : https://github.com/ufrisk/MemProcFS
Code Dokan : https://github.com/dokan-dev/dokany

### CODE REVIEW

**Review by Ana06 — COMMENTED** [review]
Could you please squash your commits (avoid `fix linter` commits in order to maintain a clean commit history). Apart from that LGTM.

File: packages/dokan.vm/dokan.vm.nuspec [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>dokan.vm</id>
+    <version>2.0.6.20231207</version>
```
Comment (Ana06) [inline packages/dokan.vm/dokan.vm.nuspec:5]: [nitpick] fourth segment is not needed

File: packages/dokan.vm/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,46 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Dokan'
+$category = 'Utilities'
+
+# Bypass linter
+$category
```
Comment (Ana06) [inline packages/dokan.vm/tools/chocolateyinstall.ps1:8]: Please add this package to the linter exclusions instead (the linter should help us writting better code and not making our code uglier 😉 ): https://github.com/mandiant/VM-Packages/blob/main/scripts/test/lint.py#L308

Sorry, I missed it the first time I reviewed it 🙈

**Review by Ana06 — CHANGES_REQUESTED** [review]
The tools seem to install & run correctly. Just one minor thing to address regarding the linter.

**Comment by naacbin** [comment]
Is there anything left to do on this PR ?

**Review by day1player — APPROVED** [review]

**Review by Ana06 — APPROVED** [review]

**Pull request merged**